content/docs/user-guide/gems/reference/aws/aws-core/configuring-credentials.md

```diff
@@ -227,6 +227,38 @@ To set up a team, repeat the instructions for individual users above to:
 
 Please read [Working with AWS credentials](#working-with-aws-credentials) to decide the right method for providing AWS IAM credentials for your O3DE project.
 
+## Running your O3DE project on Amazon EC2
+
+If you are running your project on Amazon EC2, you have the additional option of using the underlying [instance profile](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html#ec2-instance-profile) to authenticate calls made to AWS. Using the EC2 instance role lets you avoid needing to separately configure AWS credentials on the machine through less secure methods like setting the AWS environment variables through user data at deploy time or manually remoting into the machine to set up a profile.
+
+To use EC2 instance role credentials with your project:
+1. Create an EC2 [instance profile](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_use_switch-role-ec2_instance-profiles.html) through the Amazon web console or AWS CLI. An instance profile is essentially a container for an IAM role that your EC2 instance can assume to make calls to AWS.
```

Comment:
I tested this out, and launching an instance w/o setting an IAM role does not result is a role being created for the instance, it just gets spun up w/o a role
![image](https://user-images.githubusercontent.com/34254888/195197620-dbc460f6-7dce-4d9d-86f8-4bf7b198f6c8.png)

The doc at your link says that an instance profile will be created to for you if you are creating an _IAM role for an EC2 instance_, which makes sense, since you can't use a role for EC2 w/o one.

I do think it's confusing that you need two things - the instance profile and the EC2 IAM role - and sometimes they both get created at once and sometimes (e.g., AWS CLI set up) you need to make them separately. So I'll add a link to this doc from the "web console" text on this line since it sheds light on what happens during the web console workflow. What do you think?

Reply:
Yeah, I think I misunderstood what it means. Thanks for verifying it, Allie. I agree that it could be helpful to include this link the doc.